src/main/java/com/rultor/agents/ecs/StartsECS.java

```diff
@@ -41,29 +41,32 @@
 import org.xembly.Directives;
 
 /**
- * Starts Amazon EC2 instance.
+ * Starts Amazon ECS instance, that running Docker + SSHD on ECS. Instance
+ * is configured in privileged mode. Configuration is stored in S3.
```

Comment:
@xupyprmv `Starts Amazon ECS instance, that running Docker` -> `Starts Amazon ECS instance that runs Docker`